app/routes/entries/browse/$initialLetters/$pageNumber.tsx

```diff
@@ -19,11 +23,20 @@ export default function EntryDetailsPage() {
   const data = useLoaderData<typeof loader>()
   const params = useParams()
 
+  let curPage: number;
+
+  if (params.pageNumber) {
+    curPage = parseInt(params.pageNumber);
+  } else {
+    curPage = 1;
+  }
+
   return (
     <div>
       <h3 className="text-2xl font-bold">
         <>
-          Entries starting with {params.initialLetters}: {data.entries.length}
+          Entries starting with  {params.initialLetters}
+          :  {data.entries.length}  (Page {params.pageNumber})
```

Comment:
The spacing here is again something ESLint / Prettier should auto fix. When we examine your setup let's make sure it takes care of this case too.

Reply:
Done